stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -431,10 +402,18 @@ public struct TaskGroup<ChildTaskResult: Sendable> {
 // proofing, in case we'd ever have typed errors, however unlikely this may be.
 // Today the throwing task group failure is simply automatically bound to `Error`.
 
-/// A task group serves as storage for dynamically spawned, potentially throwing,
-/// child tasks.
+/// A task group serves as storage for dynamically spawned tasks,
```

Comment:
```suggestion
/// A task group serves as storage for dynamically spawned,
```

"tasks" appears later in the sequence